package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "markdown-react-js",
-  "version": "0.3.0",
```

Comment:
Sorry this is actually just a minor version bump on 0.x for the new props feature.

I was thinking the react 0.14 additions would justify a major version bump to 1.x, so that's in that branch - not clear what semver dictates about breaking peer dependencies, if you want to keep it as another minor bump I'll do that instead.